README.md

```diff
@@ -16,3 +12,50 @@ Build for your local architecture
 ```
 make releases/jumpstart
 ```
+## Getting started -
+
+#### 1. Clone the Repository
+
+```bash
+git clone https://github.com/nurdsoft/jumpstart
+```
+#### 2. Build for Local Machine
+
+Navigate to the cloned repository and build the tool for your local machine using the following command:
+```bash
+make releases/jumpstart
+```
+
+#### 3. Set Up GitHub Personal Access Token
+Create a GitHub personal access token and set it as a variable GITHUB_TOKEN in your terminal profile.
+```bash
+export GITHUB_TOKEN=<your_token>
+```
+#### 4. Generate Your Project
+Run the following command to generate a project:
+```bash
+releases/jumpstart -t <template name> <project name>
```

Comment:
How do we find what are the available templates? Is there a command for it? if yes, please list that before the generate command.
If such a command doesn't exist, we need to list all the "exact" template names somewhere.

Reply:
done